benchmark/fixtures/echo.worker.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const worker = require('worker');
```

Comment:
In case it's not been called out explicitly; no new unflagged core modules should be added until https://github.com/nodejs/TSC/issues/389 is resolved